slick/src/main/scala/slick/lifted/Query.scala

```diff
@@ -24,6 +24,12 @@ sealed abstract class Query[+E, U, C[_]] extends QueryBase[C[U]] { self =>
   def shaped: ShapedValue[_ <: E, U]
   final lazy val packed = shaped.toNode
 
+  /** Add a comment on top of the query
+    * */
+  def withComments(comment: String): Query[E, U, C] = {
```

Comment:
I mentioned in [another comment](https://github.com/Iterable/slick/pull/1/files#r1036587430) that we still need to write the code that will call this implicitly whenever our code calls `.result` or another `Query` method that generates a `DBIOAction` value. Do you want to do that in this PR, or a follow-up? If the latter, please create a ticket for it.

Reply:
I think there is already a ticket for it - https://iterable.atlassian.net/browse/FRAM-4450